xbmc/GUIInfoManager.cpp

```diff
@@ -6954,6 +6960,8 @@ bool CGUIInfoManager::GetBool(int condition1, int contextWindow, const CGUIListI
     bReturn = CProfilesManager::GetInstance().GetMasterProfile().getLockMode() != LOCK_MODE_EVERYONE;
   else if (condition == SYSTEM_HAS_PVR)
     bReturn = true;
+  else if (condition == SYSTEM_HAS_PVR_ADDON)
+    bReturn = CAddonMgr::GetInstance().HasAddons(ADDON::ADDON_PVRDLL);
```

Comment:
this should not be done here. it triggers creating instances of tons of addons on every render cycle.